docs/products/flink/concepts/kafka_connectors.rst

```diff
@@ -39,3 +39,33 @@ If you want to use multiple data streams as a source, but have the results for m
 
 In addition, choose upsert connectors if you want to provide the output as a compacted topic and read only the latest value for each message key.
 
+
+Requirements for each connector type
+------------------------------------
+
+.. note::
+
+   Aiven for Apache Flink supports the following data formats: JSON (default), Apache Avro, Confluent Avro, Debezium CDC. For more information on these, see the `Apache Flink documentation on formats <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/connectors/table/formats/overview/>`_.
+
+**Key data format**
```

Comment:
Can we make a table with the same information? IMHO It would be more easily readable

Something Like 


|Parameter| Description|Standard Connector|Upsert Connector| comments|
|----|----|----|----|----|
|**key fields**| sets the format that is used to convert the *key* part of Kafka messages|optional|required|in standard connector can be used to specify the list of keys

Reply:
True, that's now updated ;)